client/src/app/api/task/all-catagories/route.ts

```diff
@@ -0,0 +1,35 @@
+import { API_URL } from "@/config";
+import { RESULT_STATUS } from "@/constants";
+import { getallCatagoriesReturnDataType } from "@/types";
+import { NextResponse } from "next/server";
+
+
+export async function GET() {
+
+
+
+    try {
+        const getAllCatagoriesRes = await fetch(`${API_URL}/task/getallcatagories`, {
+            method: 'GET',
```

Comment:
no need to pass GET as a method name here

Reply:
lets suppose any new category is added so it wont be reflected untill you start your server again